From 3c267d5adae123ce09dd7e7fcb65da126b37aa73 Mon Sep 17 00:00:00 2001
From: "Brett T. Warden" <brett.t.warden@intel.com>
Date: Thu, 14 Nov 2019 09:11:27 -0800
Subject: [PATCH] Do not use =native

---
 configure.ac         | 3 ---
 src/arch/Makefile.am | 3 ---
 2 files changed, 6 deletions(-)

diff --git a/configure.ac b/configure.ac
index dd472be..cdf3247 100644
--- a/configure.ac
+++ b/configure.ac
@@ -129,9 +129,6 @@ AM_CONDITIONAL([AVX2_OPT], $avx2)
 AX_CHECK_COMPILE_FLAG([-msse4.1], [sse41=true], [sse41=false], [$WERROR])
 AM_CONDITIONAL([SSE41_OPT], $sse41)
 
-AX_CHECK_COMPILE_FLAG([-march=native], [arch_native=true], [arch_native=false], [$WERROR])
-AM_CONDITIONAL([MARCH_NATIVE_OPT], $arch_native)
-
 AC_ARG_WITH([extra-includes],
             [AS_HELP_STRING([--with-extra-includes=DIR],
                        [Define an additional directory for include files])],
diff --git a/src/arch/Makefile.am b/src/arch/Makefile.am
index 9e69948..e26ef2c 100644
--- a/src/arch/Makefile.am
+++ b/src/arch/Makefile.am
@@ -40,9 +40,6 @@ libtesseract_sse_la_CXXFLAGS = -msse4.1
 endif
 
 libtesseract_native_la_CXXFLAGS = -O3 -ffast-math
-if MARCH_NATIVE_OPT
-libtesseract_native_la_CXXFLAGS += -march=native -mtune=native
-endif
 libtesseract_native_la_SOURCES = dotproduct.cpp
 
 libtesseract_arch_la_SOURCES = intsimdmatrix.cpp simddetect.cpp
-- 
2.24.0

